spec/controllers/bills_controller_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+
+describe BillsController do
+
+  before(:each) { @bill = FactoryGirl.create(:bill) }
```

Comment:
I think the factories will read a little better if we abstracted away the need to specify `FactoryGirl` explicitly. Check out [this] section on the setup.

[this]: https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#configure-your-test-suite